src/sql/sqlops.d.ts

```diff
@@ -209,6 +209,31 @@ declare module 'sqlops' {
 		id: string;
 	}
 
+	/**
+ 	* Options for the actions that could happen after connecting is complete
+ 	*/
+	export interface IConnectionCompletionOptions {
+		/**
+		 * save the connection to MRU and settings (only save to setting if profile.saveProfile is set to true)
+	 	*/
+		saveTheConnection: boolean;
+
+		/**
+	 	* open the dashboard after connection is complete
+	 	*/
+		showDashboard: boolean;
```

Comment:
This should default to `false`. I don't even know if we need it to be honest, as currently Object Explorer is the only place this happens.